### PR TITLE
[ServiceBus] unskip complete message over diff recvr test for uamqp

### PR DESCRIPTION
**Pull request by swathipil**
TODO:
- [x] remove uamqp from dev reqs
- [x] fixed test_custom_endpoint_connection_verify: uamqp raises SBError not SBConnectionError

### CODE REVIEW

File: sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py [hunk 1]
```diff
@@ -2985,9 +2985,12 @@ async def test_queue_complete_message_on_different_receiver_async(self, uamqp_tr
...
                             await receiver2.complete_message(msg)
                             received_msgs.append(msg)
-                    
+
                     assert len(received_msgs) == 5
-                    
+
+                    if uamqp_transport:
+                        # wait to make sure message completed, since uamqp is taking longer in the CI
+                        asyncio.sleep(20)
```
Comment (kashifkhan) [inline sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py:2993]: missing an await here :)

Comment (swathipil) [reply sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py]: 🤦 fixing, thank you!

**Review by kashifkhan — APPROVED** [review]

**Pull request merged**